overrides.nix

```diff
@@ -779,18 +779,13 @@ self: super:
         self.setuptools-scm-git-archive
       ];
 
-      postPatch = ''
-        cat > '' +
-      (if lib.versionAtLeast super.matplotlib.version "3.5.0" then "mplsetup.cfg" else "setup.cfg") +
```

Comment:
We need to retain this if statement to be compatible with older versions of `matplotlib`, no?

Reply:
I haven't tested it, but the MPLSETUPCFG was introduced 11 years ago https://github.com/matplotlib/matplotlib/commit/6713af0624c70ac1d0c1bc42461325f1e5f80363

Reply:
@veprbl Thanks for the reference!